gripper_minimal_service/src/gripper_minimal_client_v2.cpp

```diff
@@ -10,7 +10,7 @@ class MinimalClient : public rclcpp::Node
     MinimalClient(): Node("gripper_minimal_client")
     {
       client = this->create_client<hrim_actuator_gripper_srvs::srv::ControlFinger>(
-        "/hrim_actuation_gripper_000000000004/goal");
+        "/hrim_actuator_gripper_000000000004/fingercontrol");
```

Comment:
According to the documentation this is wrong https://acutronicrobotics.com/docs/technology/hrim/model/models/actuator/gripper

Reply:
True, we need to update the docs. `fingercontrol` is the new name